deploy/kubernetes/role.yaml

```diff
@@ -44,3 +44,6 @@ rules:
 - apiGroups: [policy]
   resources: [podsecuritypolicies]
   verbs: [create, get, list, patch, update, watch, delete, use]
+- apiGroups: ['']
```

Comment:
Add new resource "events" RBAC based on changes in manifest yaml changes of adding events for ncp-patch-role.

- apiGroups:
   - ""
   resources:
     - events
   verbs:
     - create
     - get
     - list
     - patch
     - update
     - delete

Reply:
Do we need to update csv/bundle image as manifests updated?

Reply:
I think we need to do that also. we can put it in the another patch once this patched merged. I recalled that you told me we can generate bundles images from manifests automatically.